src/CopyrighFixer/CopyrighFixer/configparser.cpp

```diff
@@ -0,0 +1,66 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "configparser.h"
+#include <quasarapp.h>
+#include <QFileInfo>
+
+
+namespace CopyrighFixer {
+ConfigParser::ConfigParser() {
+
+}
+
+bool ConfigParser::parseOptions(Config &conf) const {
+
+    if (QuasarAppUtils::Params::isEndable("sourceDir")) {
+
+        QFileInfo srcDir(QuasarAppUtils::Params::getArg("sourceDir"));
+        if (srcDir.isDir()) {
+
+            conf.setSourceDir(QuasarAppUtils::Params::getArg("sourceDir"));
```

Comment:
You invoke the getArg 2 times. but you have a string path in the srcDir variable. So use the absaluteFilePath method of the QFileInfo class.
```suggestion
        QFileInfo srcDir(QuasarAppUtils::Params::getArg("sourceDir"));
        if (srcDir.isDir()) {
            conf.setSourceDir(srcDir.absoluteFilePath());
```